core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -432,6 +432,11 @@ package object config {
     ConfigBuilder("spark.dynamicAllocation.sustainedSchedulerBacklogTimeout")
       .fallbackConf(DYN_ALLOCATION_SCHEDULER_BACKLOG_TIMEOUT)
 
+  private[spark] val LEGACY_LOCALITY_WAIT_RESET =
```

Comment:
I think you said you hadn't updated all tests, but it would be nice to make sure we have one that tests the old logic as well